src/cclib/parser/psiparser.py

```diff
@@ -1009,6 +1009,59 @@ def get_normalization_factor(exp, lx, ly, lz):
             else:
                 assert self.moments[1] == dipole
 
+        ## Analytic Gradient (Psi4)
+         
+        #        -Total Gradient:
+        #   Atom            X                  Y                   Z
+        #  ------   -----------------  -----------------  -----------------
+        #     1       -0.000000000000     0.000000000000    -0.064527252292
+        #     2        0.000000000000    -0.028380539652     0.032263626146
+        #     3       -0.000000000000     0.028380539652     0.032263626146
+
+        if line.strip() == '-Total Gradient:':
+            next(inputfile)
+            next(inputfile)
+            line = next(inputfile).strip()
+            grads = []
+            while len(line) > 1:

```

Comment:
You don't have to change it, but if you didn't know, this can be replaced with
``` python
while line.strip():
```
and the strip won't be needed as part of the `next` call.